docs/proposals/release-notes.md

```diff
@@ -115,14 +115,15 @@ hundreds of entries.  The goal is to highlight the major changes for a release.
 
 The munger/bot option fits most cleanly into the existing workflow.
 
-The design will include:
+All accounting is managed on the master branch PR only.
```

Comment:
And this isn't true...   I can be, but it isn't.  cp-approved and the milestone are required on the branch as well as on master. There is a munger that is supposed to automate that:
https://github.com/kubernetes/contrib/blob/master/mungegithub/mungers/cherrypick-auto-approve.go
But bgrant didn't want PRs which were not approved on master to get LGTM and merged into the non-master branch. Which is why we required cp-approved and the milestone.

When someone does things by hand this should be important. But it 'should' be automatic (hahaha, the number of hueristics trying to match 2 PRs up is turuble)